packages/vscode-extension-bpmn-editor/src/extension/extension.ts

```diff
@@ -49,6 +49,13 @@ export function activate(context: vscode.ExtensionContext) {
             envelopePath: "dist/webview/index.js",
             resourcesPathPrefix: "dist/webview/editors/bpmn"
           }
+        ],
+        [
+          "bpmn20.xml",
+          {
+            envelopePath: "dist/webview/index.js",
```

Comment:
After https://github.com/kiegroup/kie-tooling-store/pull/64, this path changed to `"dist/webview/BpmnEditorEnvelopeApp.js"`

Reply:
I'll do the changes, but I've realized the xml structure of an Alfresco .bpmn20.xml file is a little different between your bpmn xml structure. I'll do some other research to make the extension work with those type of diagrams.

[Here](https://github.com/Alfresco/alfresco-community-repo/blob/master/repository/src/main/resources/alfresco/workflow/adhoc.bpmn20.xml) is an example of a bpmn20.xml file.

Reply:
It would be awesome if you can contribute to our codebase improving this compatibility. If you need any further help to build/compile and understand the codebase please let us know.